packages/web-components/src/components/content-block-segmented/__tests__/content-block-segmented.test.ts

```diff
@@ -0,0 +1,208 @@
+      await Promise.resolve();
+      expect(document.body.querySelector('dds-content-block-segmented')).toMatchSnapshot({ mode: 'shadow' });
+    });
+
+    it('should render with various attributes', async function() {
+      render(
+        template({
+          children: html`
+            ${image}
+            <dds-content-block-segmented-item>
+              <dds-content-group-heading>Lorem ipsum dolor sit amet.</dds-content-group-heading>
+              <dds-content-item-copy
+                >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sed interdum tortor. Sed id pellentesque diam.
+                In ut quam id mauris finibus efficitur quis ut arcu. Praesent purus turpis, venenatis eget odio et, tincidunt
+                bibendum sem. Curabitur pretium elit non blandit lobortis. Donec quis pretium odio, in dignissim sapien. Lorem
+                ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sed interdum tortor. Sed id pellentesque diam. In ut
+                quam id mauris finibus efficitur quis ut arcu. Praesent purus turpis, venenatis eget odio et, tincidunt bibendum
+                sem. Curabitur pretium elit non blandit lobortis. Donec quis pretium odio, in dignissim
+                sapien.</dds-content-item-copy
+              >
+              <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="https://example.com"
+                >Lorem Ipsum dolor sit</dds-text-cta
+              >
+            </dds-content-block-segmented-item>
+            <dds-content-block-segmented-item>
+              <dds-content-group-heading>Lorem ipsum dolor sit amet.</dds-content-group-heading>
+              <dds-content-item-copy
+                >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sed interdum tortor. Sed id pellentesque diam.
+                In ut quam id mauris finibus efficitur quis ut arcu. Praesent purus turpis, venenatis eget odio et, tincidunt
+                bibendum sem. Curabitur pretium elit non blandit lobortis. Donec quis pretium odio, in dignissim sapien. Lorem
+                ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sed interdum tortor. Sed id pellentesque diam. In ut
+                quam id mauris finibus efficitur quis ut arcu. Praesent purus turpis, venenatis eget odio et, tincidunt bibendum
+                sem. Curabitur pretium elit non blandit lobortis. Donec quis pretium odio, in dignissim
+                sapien.</dds-content-item-copy
+              >
+              ${image}
+              <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="https://example.com"
+                >Lorem Ipsum dolor sit</dds-text-cta
+              >
+            </dds-content-block-segmented-item>
+            <dds-card-cta slot="cta" cta-type="local" href="https://example.com">
+              Lorem ipsum dolor
+              <dds-card-cta-footer></dds-card-cta-footer>
+            </dds-card-cta>
```

Comment:
Wondering if this test case and similar test cases can be simpler. Note that the snapshot test in Web Components codebase generates snapshots of only Shadow DOM, not light DOM. That said, changing `children` for snapshot testing is worthwhile only if `children` should change Shadow DOM content, and thus `children` should be minimum that ensures triggering change in Shadow DOM content.

Reply:
Thanks @asudoh! I was wondering the same thing. It definitely makes sense to only render Shadow DOM. I will update :)